pages/create-collection.tsx

```diff
@@ -1,7 +1,89 @@
 import type { NextPage } from 'next';
+import { ChangeEvent, useState } from 'react';
+import Steps from '../components/common/Steps';
+import AddNftStep from '../components/createCollection/AddNftStep';
+import CollectionInfoStep from '../components/createCollection/CollectionInfoStep';
+import VerifyCollectionMint from '../components/createCollection/VerifyCollectionMint';
+import { CollectionForm, StepOneErrors } from '../utils/Interfaces';
 
-const createCollection: NextPage = function () {
-  return <div>collection flow</div>;
+const CreateNft: NextPage = function () {
+  const [step, setStep] = useState(2);
+  const [formData, setFormData] = useState<CollectionForm>({
+    tokenName: '',
+    creatorName: '',
+    displayName: '',
+    spinRoyaltiesEnabled: true,
+    royaltyWallets: [''],
+    spinPercent: 0,
+    nfts: [],
+  });
+
+  const [stepOneErrors, setStepOneErrors] = useState<StepOneErrors>({
+    tokenNameError: true,
+    creatorNameError: true,
+    displayNameError: true,
+    spinPercentError: false,
```

Comment:
"Spin" should be "split" everywhere (both in UI and code).
These conditions should not have to be fulfilled if split-royalties is disabled.
Currently, if you don't split royaltyfees in a proper way, you can't continue -- even if you don't want to split them at all, which should be valid.

Reply:
done